StarkInfra/StarkInfra/CreditNote/Invoice/Invoice.cs

```diff
@@ -138,43 +139,47 @@ internal static (string resourceName, Api.ResourceMaker resourceMaker) Resource(
 
         internal static Resource ResourceMaker(dynamic json)
         {
-            string id = json.id;
             long amount = json.amount;
             string dueString = json.due;
             DateTime? due = Checks.CheckDateTime(dueString);
             string expiration = json.expiration;
-            float? fine = json.fine;
-            float? interest = json.interest;
-            List<Discount> discounts = ParseDiscount(json.discounts);
-            List<string> tags = json.tags.ToObject<List<string>>();
+            List<string> tags = json.tags?.ToObject<List<string>>();
             List<Description> descriptions = ParseDescription(json.descriptions);
+            string id = json.id;
             string name = json.name;
             string taxId = json.taxId;
             string pdf = json.pdf;
             string link = json.link;
+            float? fine = json.fine;
+            float? interest = json.interest;
             long? nominalAmount = json.nominalAmount;
             long? fineAmount = json.fineAmount;
             long? interestAmount = json.interestAmount;
             long? discountAmount = json.discountAmount;
+            List<Discount> discounts = ParseDiscount(json.discounts);
             string brcode = json.brcode;
             string status = json.status;
             long? fee = json.fee;
             string transactionIds = json.transactionIds;
-            DateTime? created = json.created;
-            if(created != null) created = Checks.CheckDateTime(json.created);
-            DateTime? updated = json.created;
-            if (updated != null) updated = Checks.CheckDateTime(json.updated);
+            string createdString = json.created;
+            DateTime? created = Checks.CheckNullableDateTime(createdString);
```

Comment:
Essa função ja tinha no Utils e elas já faz o check se for null ela retorna null